io-engine/Cargo.toml

```diff
@@ -75,7 +75,7 @@ serde_json = "1.0.66"
 serde_yaml = "0.8.18"
 sha2 = "0.10.2"
 signal-hook = "0.3.9"
-snafu = "0.6.10"
+snafu = "0.7.1"
```

Comment:
Should we update snafu on the other crates in the workspace to avoid building two different versions?

Reply:
As I mentioned above, this version of Snafu breaks compatibility, so it might require fixes in these crates. I'll try, if it is not too much to change, I will fix it.

Reply:
It took a bit more effort that I expected but I updated anyway:
PR for crates in util/dependencies: https://github.com/openebs/mayastor-dependencies/pull/5
PR for spdk-rs: https://github.com/mayadata-io/spdk-rs/pull/12